website/docusaurus.config.ts

```diff
@@ -99,14 +99,19 @@ const config: Config = {
                     href: "/docs/introduction",
                     position: "left",
                 },
+                {
+                    to: "/typedoc",
```

Comment:
Moved this "TypeDoc" nav link next to "Documentation" - I think it makes sense to keep those grouped together.